app/models/forms/CanPayUpfrontForm.scala

```diff
@@ -27,7 +27,7 @@ object CanPayUpfrontForm {
   def form(implicit language: Language): Form[CanPayUpfrontFormValue] = {
 
     val canPayUpfrontMapping: Mapping[CanPayUpfrontFormValue] = Forms.of(EnumFormatter.format(
-      enum                    = CanPayUpfrontFormValue,
+      `enum`                  = CanPayUpfrontFormValue,
```

Comment:
Compiler made me do it, something to do with enum becoming a keyword in the future (scala 3 I think)